website/docusaurus.config.js

```diff
@@ -89,7 +96,7 @@ module.exports = {
       }
     },
     cleanUrl: true,
-    trailingSlash: true,
+    trailingSlash: false,
```

Comment:
Does this mean anybody bookmarking existing URLs (that have trailing slashes) will stop working?

Reply:
I found this setting does not work if I change it to true or false. Saying that, the existing URL will still work (with the custom script to remove the trailing slash)